wrappers/here.rb

```diff
@@ -455,13 +455,24 @@ def here_strict_restriction(param_value)
     end
 
     ##
-    # < 1000km: 15, 1500km: 10, 2000km : 5
-    def max_srcs(dist_km)
+    # motorway or toll -> dist_km < 1000km: 15, 1500km: 10, 2000km: 5
+    # !motorway or !toll -> dist_km < 1000km: 15, 1000km: 6, 2000km: 3, 3000km: 1
```

Comment:
it comment is correct, the gap is huge between 999km (15) and 1000km (6), is it correct?

Reply:
Yes it is correct. The reason is I noticed no difference under 1000km but time is drastically increased after this limit and do not change until row is at 6.

For 1000km, a 6×50 matrix with `toll=false` takes 42 secondes, any other configuration take more than 60s

Reply:
I suggest to find a way to increase value more gradually

Reply:
I have changed it and modified tests.